kubernetes/zenko/charts/redis-ha/values.yaml

```diff
@@ -6,16 +6,17 @@ enabled: true
 
 image: redis
 tag: 4.0.11-stretch
+pullPolicy: IfNotPresent
```

Comment:
This would look better for this set of values:
```yaml
image:
  repository: redis
  tag: 4.0.11-stretch
  pullPolicy: IfNotPresent
```
For reference, that's how image values look when creating a chart with `helm create` and therefore I would recommend doing so in order to keep a common format among the charts.